.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -110,6 +107,31 @@ data class CIBuildModel(
                 TestCoverage(30, TestType.vfsRetention, Os.windows, JvmCategory.MAX_VERSION.version, vendor = JvmCategory.MAX_VERSION.vendor),
                 TestCoverage(31, TestType.vfsRetention, Os.macos, JvmCategory.MIN_VERSION.version, vendor = JvmCategory.MIN_VERSION.vendor),
                 TestCoverage(32, TestType.vfsRetention, Os.macos, JvmCategory.MAX_VERSION.version, vendor = JvmCategory.MAX_VERSION.vendor))
+        ),
+        Stage(StageNames.EXPERIMENTAL_JDK14,
+            trigger = Trigger.never,
+            runsIndependent = true,
+            specificBuilds = listOf(
+                SpecificBuild.SmokeTestsJDK14,
```

Comment:
Nothing to fix right now, though it seems like we shouldn't have a class per Java version for each smoke test.